README.md

```diff
@@ -88,6 +88,42 @@ git clone git@github.com:CMSgov/beneficiary-fhir-data.git ~/workspaces/bfd/benef
     ```
 1. Install pre-commit hooks `mvn -f apps initialize`
 
+### Adding Referene to AWS Code Artifactory
+1.  In your bash_profile or your preferred shell script: add the following line to export a CodeArtifact authorization token for authorization to your repository from your preferred shell (token expires in 12 hours).  Replace {aws account id goes here} with the aws account id
+
+'''sh
+export CODEARTIFACT_AUTH_TOKEN=`aws codeartifact get-authorization-token --domain bfd-mgmt --domain-owner {aws account id goes here} --query authorizationToken --output text`
```

Comment:
This is _kind_ of ugly, but... this is basically what we're doing in CI. I figure we ought to do the same here...
```suggestion
export CODEARTIFACT_AUTH_TOKEN=`aws codeartifact get-authorization-token --domain bfd-mgmt --domain-owner "$(python3 -c 'import json,sys;print(json.loads(sys.argv[1])["Account"])' "$(aws sts get-caller-identity)")" --query authorizationToken --output text`
```

Reply:
So when I try this I get the following - > `Unknown output type: qq
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

Parameter validation failed:
Invalid length for parameter domainOwner, value: 0, valid min length: 12`

Reply:
OK. Leave well-enough alone. I think we're lacking a standard approach to aws authentication from an IC's perspective may be hurting us here. We can address this another time. Sorry for the trouble!